payments/providers/bambora_payform.py

```diff
@@ -322,16 +322,18 @@ def handle_notify_request(self):
         request = self.request
         logger.debug("Handling Bambora notify request, params: {}.".format(request.GET))
 
-        if not self.check_new_payment_authcode(request):
-            return HttpResponse(status=204)
-
         try:
             order = Order.objects.get(order_number=request.GET["ORDER_NUMBER"])
```

Comment:
Is this `ORDER_NUMBER` the `order_number+timestamp`? If so, it doesn't find the order.

Reply:
🤯 good catch